docs/aks/demo/DEMO_SCRIPT.md

```diff
@@ -0,0 +1,389 @@
+```
+
+The last line of the output will explain the proper label annotation needed when deploying the application. This will be needed later during the application install
+
+```shell
+export LABEL="<output from aad-podid.sh>"
+```
+
+## Install Helm 3
+
+Download the Helm v3 CLI:
+
+```shell
+# mac os
+OS=darwin-amd64 && \
+mkdir -p $HOME/.helm3/bin && \
+curl -sSL "https://get.helm.sh/helm-v3.0.0-${OS}.tar.gz" | tar xvz && \
+chmod +x ${OS}/helm && mv ${OS}/helm $HOME/.helm3/bin/helm3
+```
+
+or
+
+```shell
+# Linux/WSL
+OS=linux-amd64 && \
+mkdir -p $HOME/.helm3/bin && \
+curl -sSL "https://get.helm.sh/helm-v3.0.0-${OS}.tar.gz" | tar xvz && \
+chmod +x ${OS}/helm && mv ${OS}/helm $HOME/.helm3/bin/helm3
+```
+
+Add the helmv3 binary to your path and set Helm home:
+
+```shell
+export PATH=$PATH:$HOME/.helm3/bin
+export HELM_HOME=$HOME/.helm3
+```
+
+Verify the installation with:
+
+```shell
+helm3 version
+```
+
+Add the required helm repositories
+
+```shell
+helm3 repo add stable https://kubernetes-charts.storage.googleapis.com
+helm3 repo add aad-pod-identity https://raw.githubusercontent.com/Azure/aad-pod-identity/master/charts
+helm3 repo add application-gateway-kubernetes-ingress https://appgwingress.blob.core.windows.net/ingress-azure-helm-package/
+helm3 repo update
+```
+
+## Install Linkerd Service Mesh into the cluster
+
+Download the Linkerd v2 CLI:
+
+```shell
+# macOS and linux/WSL
+
+curl -sL https://run.linkerd.io/install | sh
+export PATH=$PATH:$HOME/.linkerd2/bin
+```
+
+Install the Linkerd control plane in the linkerd namespace:
+
+```shell
+linkerd install | kubectl apply -f -
+```
+
+Validate the install with:
+
+```shell
+linkerd check
+```
+
+## Install the NGNIX ingress controller
+
+Create a namespace for your ingress resources. There is a yaml file located in the clones repository under ./docs/aks/cluster/manifests/ingress-nginx-namespace.yaml
+
+```shell
+kubectl apply -f cluster/manifests/ingress-nginx-namespace.yaml
+```
+
+Use Helm to deploy an NGINX ingress controller
+
+```shell
+helm3 install ingress stable/nginx-ingress \
+    --namespace ingress-nginx \
+    --set controller.replicaCount=2 \
+    --set controller.nodeSelector."beta\.kubernetes\.io/os"=linux \
+    --set defaultBackend.nodeSelector."beta\.kubernetes\.io/os"=linux
+```
+
+## Deploy the needed componenets of helium, key rotator and the testing harness
+
+An helm chart is included for the reference application ([helium](https://github.com/RetailDevCrews/helium-csharp))
+
+Install the Helm Chart located in the cloned directory
+
+```shell
+cd docs/aks/cluster/charts
+```
+
+Create a file called helm-config.yaml with the following contents that should be edited to fit the environment being deployed in
+
+```yaml
+# Default values for bluebell.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+labels:
+  aadpodidbinding: podid #should be value of $LABEL from the output o aad-podid.sh
```

Comment:
can we use the env vars to echo this file to disk?

Reply:
Lets make this non blocking for PR and will move to M5 for cleanup